### PR TITLE
i18n(es): Update `directives-reference.mdx`, `integrations-reference.mdx` & `add-content-collections.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### Description (required)

`directives-reference.mdx` https://github.com/withastro/docs/commit/5d46e8af4bd507449fd62950c8c625e388b367c9
`integrations-reference.mdx` https://github.com/withastro/docs/commit/f7dfaca362a26e9af529030f6ca52b6acb9d08e5
`add-content-collections.mdx` https://github.com/withastro/docs/commit/7b505c36266f4e5e394fc3e229369d3506ad2725

### CODE REVIEW

File: src/content/docs/es/tutorials/add-content-collections.mdx [hunk 1]
```diff
@@ -177,11 +177,13 @@ Los siguientes pasos mostrarán cómo extender el resultado del tutorial Crear u
     };
     ```
 
+6. Para que astro reconozca tu esquema, abandona el servidor de desarrollo (`CTRL + C`) y ejecuta el siguiente comando [`npx astro sync`](/es/reference/cli-reference/#astro-sync). Esto definirá el módulo `astro:content` para la API de Colecciones de Contenido. Reinicia el servidor de desarrollo para continuar con el tutorial. 
+
```
Comment (dreyfus92) [inline src/content/docs/es/tutorials/add-content-collections.mdx:181]: aquí faltan 2 espacios, para seguir lo del commit https://github.com/withastro/docs/commit/7b505c36266f4e5e394fc3e229369d3506ad2725#diff-ec9625e9b89360bd7a29d0d392017cbb704e8890a12534044ca2374dddb9fdb5R181-R184

**Review by dreyfus92 — COMMENTED** [review]
Buen trabajo @Waxer59, te dejo una sugerencia 😁

File: src/content/docs/es/tutorials/add-content-collections.mdx (hunk 1 above)
Comment (Waxer59) [inline src/content/docs/es/tutorials/add-content-collections.mdx:181]:
```suggestion



```

Comment (Waxer59) [reply src/content/docs/es/tutorials/add-content-collections.mdx:181]: Listo! muchas gracias por la revisión 🤠

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Pull request merged**